test/controllers/folio/users/sessions_controller_test.rb

```diff
@@ -50,6 +51,92 @@ def setup
     assert_not_equal old_timestamp, user.reload.invitation_created_at
   end
 
+  test "create with crossdomain always signs in superadmins" do
+    main_site = @site
```

Comment:
```suggestion
    site_a = @site
```

To použití  "main_site" může vést k domněnce, že musí být autorizováni v `Folio.main_site`, což není pravda.
`other_site` bych pak pojmenoval `site_b`